crates/bevy_render/src/camera/projection.rs

```diff
@@ -103,12 +103,12 @@ impl CameraProjection for OrthographicProjection {
     fn update(&mut self, width: f32, height: f32) {
         match (&self.scaling_mode, &self.window_origin) {
             (ScalingMode::WindowSize, WindowOrigin::Center) => {
-                let half_width = width / 2.0;
-                let half_height = height / 2.0;
-                self.left = -half_width;
+                let half_width = (width / 2.0).round();
+                let half_height = (height / 2.0).round();
+                self.left = half_width - width;
```

Comment:
Hmm, is this about non-integer width/height due to scaling factors? I haven't really thought about those... For integer width/height, no FP inaccuracies exist. For non-integer values, my solution doesn't help anyways as far as I can tell, so we don't really need to worry about slight inaccuracies.

Reply:
Correct. However, we can examine that in a dedicated PR I suppose.

Reply:
Hmm, I wonder if there are windowing systems that could in theory support non-integral "window" dimensions. An HTML canvas maybe?

In any case, I think not rounding left/bottom is the correct thing to do, so that (right-left)==width and (top-bottom)==height still hold, which is important to make the scale accurate.